containerbuddy/config.json

```diff
@@ -1,11 +1,12 @@
 {
-  "consul": "consul:8500",
+  "consul": "{{.CONSUL}}:8500",
+  "onStart": "/opt/containerbuddy/onstart.sh",
   "services": [
     {
-      "name": "wordpress",
+      "name": "app",
```

Comment:
i'm still struggling with this a bit, I got it working by passing a json string via the compose file, but I'm not sure if that method is feasible or the best way. Do you know of any examples of doing it that way I could look over?